discretize/mixins/vtk_mod.py

```diff
@@ -138,8 +138,11 @@ class InterfaceVTK(object):
        mesh.axis_u = (1,-1,0)
        mesh.axis_v = (-1,-1,0)
        mesh.axis_w = (0,0,1)
-       # Check that the referenc fram is valid
-       mesh._validate_orientation()
+       mesh.orientation = np.array([
+           [1, -1, 0],
```

Comment:
should this be build from `axis_u`, `v`, `w`?

Reply:
I think it's better to set it each at once, so that it is guaranteed to be a valid orientation matrix, instead of assigning to each three, and then asking the user to validate them afterwards. It's possible for the mesh to get in a very odd state if they are individually set without validating them along the way.

The only other solution would be to require the user to set them explicitly in order, i.e. `axis_u`, `axis_v`, then `axis_w`.

Reply:
Although, I guess you're just specifically referring to this instance in this example...